src/include/Robot.h

```diff
@@ -20,11 +20,11 @@
 
 class Robot : public frc::TimedRobot {
  public:
-  static OI oi;
-  static DriveBase driveBase;
-  static Gripper gripper;
-  static Wrist wrist;
-  static Elevator elevator;
+  static OI * oi;
```

Comment:
If you want to initialize these in RobotInit() instead, you should use `std::unique_ptr<OI>` for the type and `std::make_unique<OI>();` to initialize it.